File: comms/bulk-messaging/senders/resolve-sender/resolve-sender.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.senders.resolve({
        recipientAddresses: [
            {
                address: "+14153902337",
                channel: "PHONE",
            },
            {
                address: "+14153902337",
                channel: "WHATSAPP",
            },
            {
                address: "davidpletnjov@example.com",
                channel: "EMAIL",
            },
        ],
    });
}
main();

File: comms/bulk-messaging/senders/resolve-sender/resolve-sender.cs
using Twilio.Comms;
using System.Threading.Tasks;
using System.Collections.Generic;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "TWILIO_ACCOUNT_SID",
            authToken: "TWILIO_AUTH_TOKEN"
        );

        await client.Senders.ResolveAsync(
            new SendersResolveRequestSenderPoolId {
                RecipientAddresses = new List<CommunicationRecipient>(){
                    new CommunicationRecipient {
                        Address = "+14153902337",
                        Channel = CommunicationChannel.Phone
                    },
                    new CommunicationRecipient {
                        Address = "+14153902337",
                        Channel = CommunicationChannel.Whatsapp
                    },
                    new CommunicationRecipient {
                        Address = "davidpletnjov@example.com",
                        Channel = CommunicationChannel.Email
                    },
                }

            }
        );
    }

}
